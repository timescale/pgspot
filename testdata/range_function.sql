SELECT * FROM pg_catalog.safe_call1(unsafe_call1());
SELECT * FROM ROWS FROM (pg_catalog.safe_call2(), unsafe_call2());
