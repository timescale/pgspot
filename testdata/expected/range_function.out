PS016: Unqualified function call: unsafe_call1 at line 1
PS016: Unqualified function call: unsafe_call2 at line 2

Errors: 0 Warnings: 2 Unknown: 0
